src/d/actor/d_a_pirate_flag.cpp

```diff
@@ -23,7 +23,8 @@ void daPirate_Flag_packet_c::setNrmVtx(cXyz*, int, int) {
 
 /* 0000083C-00000E44       .text draw__22daPirate_Flag_packet_cFv */
 BOOL daPirate_Flag_packet_c::draw() {
-    /* Nonmatching */
+    //temp return to allow to build
+    return TRUE;
```

Comment:
you shouldn't need to add returns just for it to build, it should only give a warning if a non-void function is missing a return value. if you're getting an error it's probably caused by something else.